stream-legal-entity/legal-entity-bootstrap-task/src/main/resources/application.yml

```diff
@@ -0,0 +1,303 @@
+# Stream config
+spring:
+  cloud:
+    kubernetes:
+      discovery:
+        # Set to false if using Eureka for service discovery
+        enabled: true
+  zipkin:
+    enabled: false
+  security:
+    oauth2:
+      client:
+        registration:
+          dbs:
+            authorization-grant-type: client_credentials
+            client-id: bb-client
+            client-secret: bb-secret
+            client-authentication-method: post
+        provider:
+          dbs:
+            token-uri: http://token-converter:8080/oauth/token
+backbase:
+  stream:
+    dbs:
+      access-group-presentation-base-url: http://access-control:8080
```

Comment:
it seems '/service-api/v2' is already added somewhere in the code

Reply:
True. this is now in  the generated interfaces